warehouse/internal/loadfiles/loadfiles.go

```diff
@@ -140,9 +159,14 @@ func (lf *LoadFileGenerator) ForceCreateLoadFiles(ctx context.Context, job *mode
 	return lf.createFromStaging(ctx, job, job.StagingFiles)
 }
 
+func (lf *LoadFileGenerator) AllowUploadV2JobCreation(job *model.UploadJob) bool {
+	return lf.Conf.GetBool(fmt.Sprintf("Warehouse.%s.enableV2NotifierJob", job.Upload.DestinationType), false) || lf.Conf.GetBool("Warehouse.enableV2NotifierJob", false)
```

Comment:
shouldn't this be? 
```suggestion
	return lf.Conf.GetBool(fmt.Sprintf("Warehouse.%s.enableV2NotifierJob", job.Upload.DestinationType), false) && lf.Conf.GetBool("Warehouse.enableV2NotifierJob", false)
```
 Toggle has to be enabled and enabled for a destination type. If jobv2 is false for let's say `redshift` and toggle `Warehouse.enableV2NotifierJob` is enabled, we would allow creating v2 jobs.  CMIIW

Reply:
The plan was to use `Warehouse.enableV2NotifierJob` to switch on for all destinations instead of adding flag for every destination. I never considered the case where I would like to disable it for a specific destination. With the current implementation, I'll need to have all destination flags set to true except for redshift.